src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java

```diff
@@ -50,6 +50,10 @@ public CartController(SampleService sampleService, ProjectService projectService
 	 */
 	@RequestMapping(value = {"", "/*"}, produces = MediaType.TEXT_HTML_VALUE)
 	public String getCartPage(Model model) {
+		/*
+		 * The cart is dynamically created at runtime using React.
+		 * Base file at src/main/webapp/resources/js/pages/cart/components/Cart.jsx
```

Comment:
Sorry last change, can you also add this to pages/cart.html?

Reply:
Sure, updated in f694ac950e42dcf3585cbb334d975557991b8c0b